js-intro.js

```diff
@@ -19,6 +47,14 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+const multiplyBy = (multiplied) => {
+    for(let i=0; i<myNumbers1.length; i++){
+    console.log(myNumbers1[i] * 5);
+    }
+}
+// Calling and logging the multiplication 
+
+multiplyBy()
```

Comment:
Great job with this logic! I would just use a `return` rather than `console.log()`. You may also want to save the new values * 5 to a new variable. 
Also, when calling your function with `multiplyBy()`, you'll want to pass in an argument instead of hard-coding a variable into your function. You want this function to be able to take in any array and multiply the values by 5.